packages/js-dpp/lib/errors/consensus/state/dataContract/IncompatibleDataContractSchemaError.js

```diff
@@ -0,0 +1,57 @@
+const AbstractStateError = require('../AbstractStateError');
+const Identifier = require('../../../../identifier/Identifier');
+
+class IncompatibleDataContractSchemaError extends AbstractStateError {
+  /**
+   * @param {Object} oldSchema
+   * @param {Object} newSchema
+   * @param {Error} validationError
+   * @param {Buffer|Identifier} dataContractId
+   */
+  constructor(oldSchema, newSchema, validationError, dataContractId) {
+    super(`Data Contract updated schema is not backward compatible with one defined in Data Contract with id ${Identifier.from(dataContractId)}`);
```

Comment:
How validation error is looks like? Can we add something to message to point to specific problem?

Reply:
BTW, we pass to arguments only things what we use in message. Arguments are used to recover the error on client side. If you want to add some additional info, please use setters.